packages/components/pie-cookie-banner/test/component/pie-cookie-banner.spec.ts

```diff
@@ -13,40 +15,34 @@ import {
 const englishLocale = JSON.parse(await readFile(new URL('../../locales/en-gb.json', import.meta.url)));
 const spanishLocale = JSON.parse(await readFile(new URL('../../locales/es-es.json', import.meta.url)));
 
-const componentSelector = '[data-test-id="pie-cookie-banner"]';
-const componentDescriptionSelector = '[data-test-id="banner-description"]';
-const acceptAllSelector = '[data-test-id="actions-accept-all"]';
-const necessaryOnlySelector = '[data-test-id="actions-necessary-only"]';
-const managePreferencesSelector = '[data-test-id="actions-manage-prefs"]';
-const bodyAcceptAllSelector = '[data-test-id="body-accept-all"]';
-const bodyNecessaryOnlySelector = '[data-test-id="body-necessary-only"]';
-const bodyManagePreferencesSelector = '[data-test-id="body-manage-prefs"]';
-const modalSelector = '[data-test-id="pie-modal"]';
-const modalDescriptionSelector = '[data-test-id="modal-description"]';
-const modalBackButtonSelector = '[data-test-id="modal-back-button"]';
-const modalSaveButtonSelector = '[data-test-id="modal-leading-action"]';
-const getPreferenceItemSelector = (id: PreferenceIds) => `#${id} [data-test-id="switch-component"]`;
-
 function stripTags (str: string) {
     return str.replace(/<\/?[^>]+(>|$)/g, '');
 }
 
+let pieCookieBannerComponent: PieCookieBannerComponent;
+let pieModalComponent: PieModalPage;
+
 test.describe('PieCookieBanner - Component tests', () => {
-    test('should render successfully', async ({ mount, page }) => {
+    test.beforeEach(async ({ page }) => {
+        pieCookieBannerComponent = new PieCookieBannerComponent(page);
+        pieModalComponent = new PieModalPage(page);
+    });
+
+    test('should render successfully', async ({ mount }) => {
         // Arrange
         await mount(PieCookieBanner, {
             props: {} as CookieBannerProps,
         });
 
         // Act
-        const cookieBanner = page.locator(componentSelector);
+        const cookieBanner = await pieCookieBannerComponent.isCookieBannerVisible();
 
         // Assert
-        expect(cookieBanner).toBeVisible();
+        expect(cookieBanner).toBeTruthy();
```

Comment:
can this not be `toBe(true)`? is a bit more explicit

Reply:
Yeah, playwright's native , `toBeTrue` uses truthy instead of true but Truthy is already an explicit command?

`Value is truthy, i.e. not false, 0, null, etc.`

Reply:
so the result could be `''` instead of `true`?